### ISSUE TITLE
Backend-Event code request

### ISSUE THREAD

**Issue by TheTime000** [ad410project/ad410project#59]
Is it ideal to write some function code in the events.php ... Or are we keeping this way for now?

**Comment by Fremonster** [comment]
Yes, we need to work on filling in the code for all the functions in the events class. The stored procedures file (procedures_db.sql) in the Models folder has some procedures that can be used to get information from the database. Either those can be used (http://www.mysqltutorial.org/php-calling-mysql-stored-procedures/) or you can write your own sql queries like we did last quarter. Thanks!
